src/utils/directory.js

```diff
@@ -21,9 +21,14 @@ const ensureDirectoryPathExists = (pathToFolder) => {
   }
 };
 
+const shouldIgnorePath = (path, ignoredFolders) => ignoredFolders.some(
+  (folder) => path.indexOf(folder) > -1
```

Comment:
this might filter some unexpected directories, for example if the path is "testing/this/path" and the ignored directory is "test" then this will be filtered.

we could use the os safe path [split](https://nodejs.org/api/path.html#path_path_sep) then check each directory for an exact match 

```
const path = require('path')

const shouldIgnorePath = (path, ignoredFolders) => ignoredFolders.some(
  (folder) => path.directory(filePath).split(path.sep).some(dir => dir === ignoredFolders)
);
```

Reply:
Great suggestion, that worked beautifully. I also added an additional test case for the scenario you described.